scripts/app.js

```diff
@@ -0,0 +1,105 @@
+const getBooks = () => {
+  let books;
+  if (localStorage.getItem('books') === null) {
+    books = [];
+  } else {
+    books = JSON.parse(localStorage.getItem('books'));
+  }
+
+  return books;
+};
+
+const addBookToList = (book) => {
+  const bookList = document.getElementById('booklist');
+
+  const div = document.createElement('div');
+
+  div.innerHTML = `
+    <p>${book.title}</p>
+    <p>${book.author}</p>
+    <p>${book.isbn}</p>
+    <button class="btn delete">Remove Book</button>
+    `;
+
+  bookList.appendChild(div);
+};
+
+// Store Functions: Handles Storage
+
+const addBook = (book) => {
+  const books = getBooks();
+
+  books.push(book);
+
+  localStorage.setItem('books', JSON.stringify(books));
+};
+
+const removeBook = (isbn) => {
+  const books = getBooks();
+
+  books.forEach((book, index) => {
+    if (book.isbn === isbn) {
+      books.splice(index, 1);
+    }
+  });
```

Comment:
- [ ] I would recommend you to use `.filter()` function instead of `.forEach()` as the project requirements suggest 😄

Reply:
This issue has been addressed well. Nice job! :clap: